.env.example

```diff
@@ -1,5 +1,3 @@
 ETHERSCAN_API_KEY=ABC123ABC123ABC123ABC123ABC123ABC1
-ROPSTEN_URL=https://eth-ropsten.alchemyapi.io/v2/<YOUR ALCHEMY KEY>
 PRIVATE_KEY=0xabc123abc123abc123abc123abc123abc123abc123abc123abc123abc123abc1
-FORK_MAINNET=
-QUICK_NODE_KEY=
```

Comment:
I think we should keep the variables in the env example file. If we dont want to run fork tests at a momwnt simply we can pass `FORK_MAINNET` as false in env to do that.